website/forms/php/filters/general/set-locale.md

```diff
@@ -3,4 +3,11 @@ id: set-locale
 title: Set locale
 ---
 
-*Coming soon*
+This filter can be used to change the value of current locale. By default WordPress sets the locale in the admin to `en_US` and with this filter it can be changed to whichever locale is needed (e.g. when using multilanguage plugin).
+
+```php
+add_filter('es_forms_general_set_locale', function(string $locale): string {
+	// Get the custom locale (e.g. from WPML plugin).
+	return $locale;
+})
```

Comment:
```suggestion
This filter can be used to change the value of current locale. By default, WordPress sets the locale in the admin to `en_US`. With this filter it can be changed to any locale (e.g. when using a multi-language plugin).

```php
add_filter('es_forms_general_set_locale', function(string $locale): string {
	// Get the custom locale (e.g. from the WPML plugin).
	return $locale;
})
```